_includes/releases/v22.2/v22.2.7.md

```diff
@@ -0,0 +1,147 @@
+
+<h3 id="v22-2-7-command-line-changes">Command-line changes</h3>
+
+- [Workloads](../v22.2/cockroach-workload.html) that take a `--seed` argument used to default to `1`. Now, they use a randomly generated seed in each run. Users can still pass a custom seed with the `--seed` flag. [#95695][#95695]
+- The `--drain-wait` argument to the [`cockroach node drain`](../v22.2/cockroach-node.html) command will be automatically increased if the command detects that it is smaller than the sum of the [cluster settings](../v22.2/node-shutdown.html#cluster-settings) `server.shutdown.drain_wait`, `server.shutdown.connection_wait`, `server.shutdown.query_wait` times two, and `server.shutdown.lease_transfer_wait`. If the `--drain-wait argument` is `0`, then no timeout is used. [#98577][#98577]
+
+<h3 id="v22-2-7-db-console-changes">DB Console changes</h3>
+
+- Fixed the error `Cannot read properties of undefined (reading 'length')` which can cause [DB Console](../v22.2/ui-overview.html) pages fail to load. [#98236][#98236]
+- Updated the column selector icon. [#98915][#98915]
+- Updated the Jobs table column name to the correct Last Execution Time. [#99036][#99036]
+
+<h3 id="v22-2-7-bug-fixes">Bug fixes</h3>
+
+- Fixed a bug introduced in [v22.1](../releases/v22.1.html) that caused the internal error `no bytes in account to release ...`. [#97773][#97773]
+- Fixed a bug that caused an internal error when trying to execute a [UDF](../v22.2/user-defined-functions.html) with an empty function body. This bug was present since UDFs were introduced in [v22.2.0](../releases/v22.2.html). [#93834][#93834]
+- Fixed the [`SHOW CREATE ALL {TYPES|SCHEMAS|TABLES}`](../v22.2/show-create.html) commands so they handle database names that have mixed-case, hyphens, or quotes. [#97937][#97937]
+- Fixed link encoding on links to database/table/index pages. [#97930][#97930]
+- The Jobs page now displays an error state when an error occurs during data fetching. [#97723][#97723]
+- Changed the database used for SQL API calls to no longer use `defaultdb`, which was causing error messages on some pages when that database no longer exists. [#98052][#98052]
+- The owner of the public schema can now be changed using [`ALTER SCHEMA public OWNER TO new_owner`](../v22.2/alter-schema.html). [#98065][#98065]
+- Fixed a bug where the experimental scrub command did not handle type descriptors in the database. [#91459][#91459]
+- Fixed a bug in which [common table expressions (CTEs)](../v22.2/common-table-expressions.html) marked as `WITH RECURSIVE` which were not actually recursive could return incorrect results. This could happen if the CTE used a [`UNION ALL`](../v22.2/common-table-expressions.html#recursive-common-table-expressions), because the optimizer incorrectly converted the `UNION ALL` to a `UNION`. This bug has existed since suppport for recursive CTEs was first added in [v20.1](../releases/v20.1.html). [#98115][#98115]
```

Comment:
Maybe:

Fixed a bug where [common table expressions (CTEs)](../v22.2/common-table-expressions.html) marked as `WITH RECURSIVE` that were not actually recursive could return incorrect results.